Library/Formula/dropbox-uploader.rb

```diff
@@ -0,0 +1,15 @@
+class DropboxUploader < Formula
+  desc "Bash script for interacting with Dropbox"
+  homepage "http://www.andreafabrizi.it/?dropbox_uploader"
+  url "https://github.com/andreafabrizi/Dropbox-Uploader/archive/0.16.tar.gz"
+  sha256 "738aebdb57f75e9033cb9bb319b14fe1c3b471f951180acb48631e9810ebf1c3"
+
+  def install
+    bin.install "dropbox_uploader.sh"
```

Comment:
It’s probably better to have it named `dropbox_uploader` (without the extension). To install it under this name use:
```rb
bin.install "dropbox_uploader.sh" => "dropbox_uploader"
```